lib/chat_api_web/channels/notification_channel.ex

```diff
@@ -56,6 +58,14 @@ defmodule ChatApiWeb.NotificationChannel do
       message = Messages.get_message!(message.id)
       result = ChatApiWeb.MessageView.render("expanded.json", message: message)
 
+      # If the received message is the first one sent by any company user, assign the conversation to the user
+      if Messages.count_messages_sent_by_company_user_in_conversation(message.conversation_id) ==
```

Comment:
hmm I'm tempted to just encapsulate this logic in a method like
```ex
Conversations.has_company_user_replied?(conversation_id)
```
which would basically just check if the result of the count query is `> 0`... not sure that `@messages_limit_to_autoassign` is necessary at the moment

what do you think?